jobs/build/refresh-images/Jenkinsfile

```diff
@@ -112,14 +135,76 @@ images:update-dockerfile
   --push
   """
 
-                buildlib.oit """
+//
+// start retry region
+//
+                BUILD_CONTINUED = false
+                waitUntil {
+                    try {
+                        exclude = ""
+                        if (BUILD_EXCLUSIONS != "") {
+                            exclude = "-x ${BUILD_EXCLUSIONS} --ignore-missing-base"
+                        }
+
+                        buildlib.oit """
 --working-dir ${OIT_WORKING} --group openshift-${OSE_MAJOR}.${OSE_MINOR}
 images:build
 --push-to-defaults --repo-type signed
 """
+                        return true  // finish waitUntil
+                    }
+
+                    catch (err) {
+                        failed_builds = get_failed_builds(OIT_WORKING)
+
+                        mail(
+                            to: "${MAIL_LIST_FAILURE}",
+                            from: "aos-cicd@redhat.com",
+                            subject: "RESUMABLE Error during Refresh Images for OCP v${BUILD_VERSION}",
```

Comment:
BUILD_VERSION needs to be initialized somewhere. It will just be BUILD_VERSION="${OSE_MAJOR}.${OSE_MINOR}"